.changeset/swift-rabbits-appear.md

```diff
@@ -0,0 +1,71 @@
+---
+"@latticexyz/explorer": patch
+---
+
+Initial release of the @latticexyz/explorer package. Explorer is a standalone tool designed to explore and manage worlds. This initial release supports local worlds, with plans to extend support to any world in the future.
+
+## Using explorer
```

Comment:
I think everything below this line should be in the README instead of the changeset. We can point to the README or docs from the changeset.

Reply:
totally, should have thought about that, adjusted the README.md with a link to it from the changeset